asciidoc-confluence-publisher-client/src/test/java/org/sahli/asciidoc/confluence/publisher/client/http/ConfluenceRestClientTest.java

```diff
@@ -164,14 +166,15 @@ public void getPageByTitle_withMultipleResults_throwsMultipleResultsException()
     @Test
     public void addAttachment_withValidParameters_sendsMultipartHttpPostRequest() throws Exception {
         // arrange
-        CloseableHttpClient httpClientMock = recordHttpClientForSingleResponseWithContentAndStatusCode("", 200);
-        ConfluenceRestClient confluenceRestClient = new ConfluenceRestClient(CONFLUENCE_ROOT_URL, httpClientMock, null, null);
+        ConfluenceServer.postAttachmentSucceeds("1234");
```

Comment:
Do you see a way to keep the existing test approach also in this test? If so, wiremock and the `ConfluenceServer` class would become obsolete and all tests follow a consistent approach.

Reply:
Current tests mock Apache Http Client. Wiremock gives an ability to test the validity of requests sent via http. It also gives possibility to test error cases / timeouts / retries. If there is an issue that other tests are done using another approach -- I can rework them as well.

Reply:
Feel free to propose further improvements and make the approach consistent across the whole code base - but please keep in mind that large pull requests with several unrelated changes are very time-consuming to review and integrate and thus, are often left behind due to time constraints on the repository owner side. Therefore, we prefer small, focussed pull request that deliver functionality and avoid major refactorings (please also see https://github.com/confluence-publisher/confluence-publisher/blob/master/CONTRIBUTING.md)